libr/bin/format/mach0/mach0.c

```diff
@@ -2095,16 +2084,9 @@ void *MACH0_(mach0_free)(struct MACH0_(obj_t) *mo) {
 }
 
 void MACH0_(opts_set_default)(struct MACH0_(opts_t) *options, RBinFile *bf) {
-	if (!options) {
-		return;
-	}
-
+	r_return_if_fail (options && bf);
 	options->header_at = 0;
-	if (bf && bf->rbin) {
-		options->verbose = bf->rbin->verbose;
-	} else {
-		options->verbose = false;
-	}
+	options->verbose = bf && bf->rbin && bf->rbin->verbose;
```

Comment:
no need to check `bf`, it is not-null for sure at this point

Reply:
actually, we don't even need to check if bf->rbin exist, because RBinFile should always be created with a reference to the RBin.